### ISSUE TITLE
Word frequencies in TransfoXLTokenizer

### ISSUE THREAD

**Issue by GregoireMialon** [huggingface/transformers#5906]
I was wondering if it is still possible to access word frequencies through the populated counter of TransfoXLTokenizer? For example, `tokenizer = TransfoXLTokenizer.from_pretrained('transfo-xl-wt103')` seems to have an empty counter. This is referring to:

"So p_M(S) is just the output of the model right?

For p_u(S), I think the easiest is probably to use the empirical probabilities.
`TransfoXLTokenizer` has a counter to store words frequencies [here](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/tokenization_transfo_xl.py#L98) which should be populated in the "pretrained" tokenizer so I would use and normalize this to get unconditional probabilities for each word and then compute SLOR."

_Originally posted by @thomwolf in https://github.com/huggingface/transformers/issues/477#issuecomment-483973033_
